src/Collectors/MethodCallCollector.php

```diff
@@ -56,22 +56,25 @@ public function processNode(Node $node, Scope $scope): ?array
             return null;
         }
 
-        $classMethodCallReference = $this->classMethodCallReferenceResolver->resolve($node, $scope);
-        if (! $classMethodCallReference instanceof MethodCallReference) {
-            return null;
-        }
+        $classMethodReferences = [];
 
-        $className = $classMethodCallReference->getClass();
-        $methodName = $classMethodCallReference->getMethod();
+        $classMethodCallReferences = $this->classMethodCallReferenceResolver->resolve($node, $scope);
+        foreach($classMethodCallReferences as $classMethodCallReference) {
```

Comment:
just put a loop around the pre-existing checks to handle the newly returned iterable type from `resolve`